### PR TITLE
Windows getting started

### PR DESCRIPTION
**Pull request by StarfallProjects**

### CODE REVIEW

File: pages/agent/v3/windows.md.erb [hunk 1]
```diff
@@ -4,42 +4,36 @@ The Buildkite Agent is supported on Windows 8, Windows Server 2012, and newer.
...
-1. Open the Start menu and type "powershell"
-2. Highlight "Windows Powershell" in the results
-3. Press CTRL+SHIFT+ENTER (you'll be prompted to allow this app to make changes)
-
-Once you're in an escalated Powershell session, you can run this script to install the latest version of the Agent:
-
-```
-$env:buildkiteAgentToken = "my_token"
+```shell
+$env:buildkiteAgentToken = "<your_token>"
```
Comment (plaindocs) [inline pages/agent/v3/windows.md.erb:14]: Does Windows have a typical command prompt? And would it be useful to include it?  I can't tell how many commands this is.

File: pages/agent/v3/windows.md.erb [hunk 2]
```diff
@@ -50,6 +44,23 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
...
+
+* Set your [agent token](/docs/agent/v3/tokens), if you did not set it as an environment variable during installation.
+* You may need to use the `shell` configuration option. On Windows, Buildkite defaults to using Batch. If you want to use PowerShell or Bash, you must point Buildkite to the correct shell. For example, to use PowerShell:
+
+    ```cfg
+    shell="<path-to-powershell-executable>"
+    ```
+
+    <div class="Docs__note">
+    <p>This allows you to run a PowerShell script, but your build hooks must still be either `.bat` or `.sh` files (`.sh` files require access to Git for Windows). Refer to <a href="https://buildkite.com/docs/agent/v3/hooks#hooks-on-windows">Hooks - Hooks on Windows</a> for more information.</p>
```
Comment (plaindocs) [inline pages/agent/v3/windows.md.erb:57]: In great sadness, thse markdown `code` need to be HTML cos they are inside the note.

That is a thing I want to fix.

File: pages/agent/v3/windows.md.erb [hunk 3]
```diff
@@ -50,6 +44,23 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
...
+    </div>
+
+    <div class="Docs__note">
+    <p>Buildkite does not support the cross-platform edition of PowerShell (PowerShell 6 and 7, also knows as PowerShell Core). Make sure `shell` points to a `powershell.exe` file, not `pwsh.exe`.</p>
```
Comment (plaindocs) [inline pages/agent/v3/windows.md.erb:61]:
```suggestion
    <p>Buildkite does not support the cross-platform edition of PowerShell (PowerShell 6 and 7, also known as PowerShell Core). Make sure `shell` points to a `powershell.exe` file, not `pwsh.exe`.</p>
```

File: pages/tutorials/getting_started.md.erb [hunk 4]
```diff
@@ -8,33 +8,39 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
...
 
-[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that will be run on your code.
-
-We’ve created a [bash-example test repository](https://github.com/buildkite/bash-example) to use as your first pipeline. Click the “Add to Buildkite” button below to open a new pipeline page for the example pipeline:
-
-<a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
+[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that you run on your code.
 
-Accept the pre-filled defaults, and click *Create Pipeline*.
+We've created sample pipelines to use as your first pipeline:
```
Comment (plaindocs) [inline pages/tutorials/getting_started.md.erb:19]: Let's avoid using "we" here (note to self, lint that)

File: pages/tutorials/getting_started.md.erb [hunk 5]
```diff
@@ -8,33 +8,39 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
...
 
-Type in a build message (being sure to add some [emojis](/docs/pipelines/emojis)) and click *Create Build*:
+    [powershell-example test repository](https://github.com/buildkite/powershell-example)
 
-<%= image "trigger-build.png", size: "600x460", alt: "New build dialog" %>
+    <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/powershell-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add PowerShell Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-The build will run, and once it’s finished it’ll turn green!
+2. Accept the pre-filled defaults, and select *Create Pipeline*.
+3. The example repositories contain a `pipeline.yml` setting out the pipeline steps, so on the *Steps* screen, add the following to upload and use the `pipeline.yml`:
```
Comment (plaindocs) [inline pages/tutorials/getting_started.md.erb:32]: I have mixed feelings about this switch. It is definitely an improvment, but it is gonna break if folks *don't* do the YAML pipelines step.

**Review by plaindocs — CHANGES_REQUESTED** [review]
This is great, and we're pretty close to getting this done!

File: pages/agent/v3/windows.md.erb (hunk 1 above)
Comment (StarfallProjects) [reply pages/agent/v3/windows.md.erb:14]: Generally no. Looking at PowerShell's own docs, they're a bit inconsistent themselves. Some have nothing, some have an example path, and some have PS>. However PS> will make no sense for someone using command prompt. And > will make no sense for someone using Git Bash, which Buildkite encourages.

But this is a PowerShell specific example, so I've gone with PS>

File: pages/tutorials/getting_started.md.erb (hunk 5 above)
Comment (StarfallProjects) [reply pages/tutorials/getting_started.md.erb:32]: Yes. However I've tried to signal clearly with the note, and I really don't think we should be using the deprecated method. Having several ways of handling pipeline config is messy, and having the default for new accounts be the less-favoured way is not ideal, but hopefully the docs somewhat address that if they steer people to the new way from the get-go.

File: pages/agent/v3/windows.md.erb [hunk 6]
```diff
@@ -4,42 +4,36 @@ The Buildkite Agent is supported on Windows 8, Windows Server 2012, and newer.
...
-$env:buildkiteAgentToken = "my_token"
-Set-ExecutionPolicy Bypass -Scope Process -Force
+```shell
+PS> $env:buildkiteAgentToken = "<your_token>"
+PS> Set-ExecutionPolicy Bypass -Scope Process -Force
 iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/buildkite/agent/main/install.ps1'))
 ```
 
-## Manual Installation
+## Manual installation
```
Comment (plaindocs) [inline pages/agent/v3/windows.md.erb:19]: Do you need admin privileges to run this? Do we prefer one over the other? At the mo it isn't super clear that you don't need to run both. Maybe a short line to the effect of

> If you can't install using the autaomtic method you can use this one?

File: pages/tutorials/getting_started.md.erb [hunk 7]
```diff
@@ -8,33 +8,39 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
...
+[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that you run on your code.
 
-Accept the pre-filled defaults, and click *Create Pipeline*.
+Choose a sample pipeline to use as your first pipeline:
 
-## Run your first build
+1. Click the appropriate *Add to Buildkite* button:
 
-Now that you’re running an agent and have created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
+    [bash-example test repository](https://github.com/buildkite/bash-example)
```
Comment (plaindocs) [inline pages/tutorials/getting_started.md.erb:23]: Do you think we need an explicit if you're running a Windows agent, or if you're running anything else type text?

Comment (StarfallProjects) [reply pages/tutorials/getting_started.md.erb:23]: The steps above mention they need to start the agent. I've added a bit to that section to tell them how to confirm the agent is running and linked to their org correctly.

This is universal, whatever they're on. In theory, they could use the bash example repo on windows, IF they can point to git bash successfully (which admittedly I could not) . . .

File: pages/agent/v3/windows.md.erb (hunk 6 above)
Comment (StarfallProjects) [reply pages/agent/v3/windows.md.erb:19]: Manual installation doesn't run anything, you're literally just copying files around. They might need admin privileges depending on where they decide to copy them to/any restrictions on their computer, but that really is into "can they manage their own machine" territory. I've added a sentence up top to make clear there are two installation methods.

**Review by plaindocs — APPROVED** [review]
LGTM!

**Pull request merged**